tiled/client/context.py

```diff
@@ -6,11 +6,14 @@
 import urllib.parse
 import warnings
 from pathlib import Path
+from typing import List
 from urllib.parse import parse_qs, urlparse
 
 import httpx
 import platformdirs
 
+from tiled.server.schemas import AboutAuthenticationProvider
```

Comment:
This pulls in server-side deps. This keeps coming up; maybe we should move these schemas into `tiled.schemas`.